common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -220,6 +220,9 @@ public static Serializable base64StringToSerializable(String input) throws IOExc
     }
 
     public static Map<String, byte[]> mapOfBase64StringToByteArray(Map<String, String> input) {
+        if (input == null) {
+            return null;
```

Comment:
I don't know what the others think, but what about returning an empty map to prevent further NPE?

Reply:
@jrochas I thought the same, I also thought of initializing the propagatedVariables map of TaskResultImpl (further down in my stack) to an empty map from the beginning as it's the only usage of the method. Then I saw that other methods in the class also had this `return null` (maybe for a purpose, I didn't check them all tbh).

Reply:
@jrochas in your opinion should we return empty objects instead of null in all cases?